### ISSUE TITLE
Email includes wrong path to job

### ISSUE THREAD

**Issue by sbliven** [eppic-team/eppic#84]
After submitting a user job in eppic3, You get an email like

> 1qdm.cif.gz job submitted.
> To follow the job's status you can check it at the URL:
> http://pc11467:8081/ewui/#id/Enju11sLNE4YbJbELYR5jl3SoUsUYj
> ...

This URL is wrong; it should not contain the 'ewui' part.

**Comment by sbliven** [comment]
BTW, the URL comes from the 'resultsPathURL' member of CrkWebServiceImpl, which is set in init() to the servletContPath. Should that perhaps be empty in eppic3?

**Comment by josemduarte** [comment]
The issue here is that we've moved the webapp from the ewui namespace to the root namespace. I did that unintentionally when upgrading to jetty 9 because I didn't find a way to redirect the root webapp to the ewui webapp and instead had to set up the ewui webapp as the root (which then removes the `ewui` prefix).

I think we should really try to keep URLs as they used to be, keeping the `ewui` prefix. Hitting the root webapp should then redirect to ewui as we do in current production. If we manage to do that then this issue would also disappear.
